pkg/controller/controller_serviceplan.go

```diff
@@ -35,17 +34,7 @@ func (c *controller) servicePlanAdd(obj interface{}) {
 }
 
 func (c *controller) reconcileServicePlanKey(key string) error {
-	servicePlan, err := c.servicePlanLister.Get(key)
-	if errors.IsNotFound(err) {
-		glog.Infof("Not doing work for ServicePlan %v because it has been deleted", key)
-		return nil
-	}
-	if err != nil {
-		glog.Errorf("Unable to retrieve ServicePlan %v from store: %v", key, err)
-		return err
-	}
-
-	return c.reconcileServicePlan(servicePlan)
+	return nil
```

Comment:
Can we add a comment here stating that service plans are reconciled as part of reconciling service brokers? It was not immediately clear to me why we were not doing anything in this function.

Reply:
Yes; I was actually very surprised to see a control loop calling this function.